### ISSUE TITLE
SMOKEVIEW - Nozzles are displayed in wrong direction

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#1045]
```
Application Version: SMV 5.5.3
SVN Revision Number: 5998
Compile Date: April 6, 2010
Operating System: Windows 7, 64 bit

I was running a job with various nozzles a few days ago using FDS 5.4.3.,
MPI enabled. Everything worked fine.
However, after installing FDS 5.5 and SMV 5.5.3 and watching the results in
smokeview again, I noticed that all the nozzles are displayed in the wrong
direction. It does not have any effect on the calculation, so I believe
that it is just a smokeview problem.
Anyways, I made a couple more very simple testcases with only one single
nozzle pointing in the positive x-direction to make sure. Whether I ran
them with FDS 5.4.3 or 5.5., the nozzles are displayed correctly with SMV
5.4.8 and displayed in the negative x-direction with SMV 5.5.3.

Best regards,
Tobias... 

```

Original issue reported on code.google.com by `t.meyer@insa4.de` on 2010-04-22 07:48:00

**Comment by gforney** [comment]
```
This appears to be a duplicate of #1070.  Merging in to that issue to clean up the
tracker list.
```

Original issue reported on code.google.com by `drjfloyd` on 2010-05-06 12:41:20
- **Labels added**: Application-SMV

**Comment by gforney** [comment]
```
this was fixed in smokeview 5.5.4 which is available on the download site
```

Original issue reported on code.google.com by `gforney` on 2010-05-06 12:54:08
